### PR TITLE
Revival surgery grabs ghosts

### PR DESCRIPTION
**Pull request by Maurukas**
## About The Pull Request

Slightly modifies the revival surgery to stuff ghosts back in their bodies when used. I believe #49090 made this the default behavior for defibs. This is small and gets it off @Jared-Fogle's to-do list.

## Why It's Good For The Game

Consistency.

## Changelog
:cl:
tweak: Revival surgery now puts ghosts back in bodies
/:cl:

### CODE REVIEW

**Comment by Jared-Fogle** [comment]
Can you lower the time it takes too? We did this for defibs too when they started grabbing ghosts. It's only a long time to give time for the ghost to come back to their body.

File: code/modules/surgery/revival.dm [hunk 1]
```diff
@@ -29,7 +29,7 @@
 	name = "shock body"
 	implements = list(/obj/item/shockpaddles = 100, /obj/item/melee/baton = 75, /obj/item/gun/energy = 60)
 	repeatable = TRUE
-	time = 120
+	time = 50
```
Comment (Rohesie) [inline code/modules/surgery/revival.dm:32]:
```suggestion
	time = 5 SECONDS
```

Comment (Maurukas) [reply code/modules/surgery/revival.dm]: There's quite a few other surgeries still using deciseconds. Should they be cleaned up as well?

Comment (Rohesie) [reply code/modules/surgery/revival.dm]: Yes

**Review by Rohesie — APPROVED** [review]

**Pull request merged**